API.php

```diff
@@ -1174,6 +1175,7 @@ public function enablePreviewMode($idSite, $idContainer, $idContainerVersion = n
 
         $cookie = new PreviewCookie();
         $cookie->enable($idSite, $idContainer);
+        $cookie->enableDebugSiteUrl(Site::getMainUrlFor($idSite));
```

Comment:
@AltamashShaikh could we do this similar to mentioned in https://github.com/matomo-org/tag-manager/pull/355#discussion_r706949800 where we do this only if there is one domain configured?

Reply:
As discussed we are removing this and will let the user to enter the debug url